test/linalg/symmetric.jl

```diff
@@ -248,10 +248,12 @@ let A = [1.0+im 2.0; 2.0 0.0]
     @test_throws ArgumentError Hermitian(A)
 end
 
-# Unary minus for Symmetric matrices
-let A = Symmetric(randn(5,5))
-    B = -A
-    @test A + B ≈ zeros(5,5)
+# Unary minus for Symmetric/Hermitian matrices
+let A = randn(5, 5)
+    for SH in (Symmetric(A), Hermitian(A))
+        F = Matrix(SH)
+        @test -SH::typeof(SH) ≈ -F
```

Comment:
I'll try again here instead of in a two years old commit. You should use `==` here.